src/js/messaging/containers/Compose.jsx

```diff
@@ -100,6 +105,22 @@ class Compose extends React.Component {
     this.props.deleteComposeMessage();
   }
 
+  handleAttachmentsChange(domEvent) {
```

Comment:
As we're adding more handlers for the various form components, I've been thinking it might be more organized to have the form components handle the DOM events themselves internally and pass the relevant info. Like in this case, this handler would only need to know about the list of files, and MessageAttachments could take care of pulling that data out from the DOM event. Out of the scope of this PR, but something to consider later.